va/validation-authority_test.go

```diff
@@ -308,18 +306,6 @@ func TestSimpleHttp(t *testing.T) {
 	test.AssertEquals(t, invalidChall.Error.Type, core.UnknownHostProblem)
 	va.TestMode = true
 
-	chall.Path = pathUnsafe
-	invalidChall, err = va.validateSimpleHTTP(ident, chall)
-	test.AssertEquals(t, invalidChall.Status, core.StatusInvalid)
-	test.AssertError(t, err, "Path doesn't consist of URL-safe characters.")
-	test.AssertEquals(t, invalidChall.Error.Type, core.MalformedProblem)
-
-	chall.Path = pathUnsafe302
-	invalidChall, err = va.validateSimpleHTTP(ident, chall)
-	test.AssertEquals(t, invalidChall.Status, core.StatusInvalid)
-	test.AssertError(t, err, "Redirect should have failed.")
-	test.AssertEquals(t, invalidChall.Error.Type, core.ConnectionProblem)
-
```

Comment:
Should these updated to suit, instead of removed? The new DNS lookup code looks like it follows redirects incorrectly by sending the redirect target URL to the original host's IP address. Adding some names like localhost2=127.0.0.2 to the DNS mock should make this testable.

Related: When redirecting, should an IP address be logged for each HTTP request performed?

Logging is testable now. :)